follej/session09/Circle.py

```diff
@@ -0,0 +1,101 @@
+from math import pi, sqrt
+
+
+class Circle(object):
+    def __init__(self, radius):
+        self._radius = radius
+
+    @property
+    def radius(self):
+        return self._radius
+
+    @radius.setter
+    def radius(self, radius):
+        self._radius = radius
+
+    @property
+    def diameter(self):
+        return self._radius * 2
+
+    @diameter.setter
+    def diameter(self, diameter):
+        self._radius = diameter / 2
+
+    @staticmethod
+    def radius_to_area(radius):
+        return pi * radius ** 2
+
+    @staticmethod
+    def radius_to_perimeter(radius):
+        return pi * radius * 2
+
+    @property
+    def area(self):
+        return Circle.radius_to_area(self.radius)
```

Comment:
you should be able to do `self.radius_to_area(self.radius)` here rather than using class name.